### DISCUSSION TITLE
How do I start opengothic?

### DISCUSSION THREAD

**Discussion by bioblatt** [Try/OpenGothic#264] [Q&A]
I've installed it on my Arch Linux from the arch user repository AUR.
Which command in the terminal starts opengothic? 
There's no entry in my application menu and typing "opengothic" in the terminal does nothing at all.

**Comment by Try** [comment]
Hi, @bioblatt !
Not quite sure about AUR - they do they own packaging.
Assume that it should be similar to how it runs, if compiled manually:
`./Gothic2Notr -g "path/to/Steam/steamapps/common/Gothic II"`

**Reply by bioblatt** [reply]
Thank you, but it failed after showing the start screen for a second.
Worked after compiling the newest build by myself :)
